android.sh

```diff
@@ -53,15 +53,18 @@ LIBRARY_EXPAT=41
 LIBRARY_SNDFILE=42
 LIBRARY_LEPTONICA=43
 LIBRARY_LIBSAMPLERATE=44
-LIBRARY_ZLIB=45
-LIBRARY_MEDIA_CODEC=46
+LIBRARY_CPU_FEATURES=45
+LIBRARY_ZLIB=46
+LIBRARY_MEDIA_CODEC=47
 
 # ENABLE ARCH
 ENABLED_ARCHITECTURES=(1 1 1 1 1)
 
 # ENABLE LIBRARIES
 ENABLED_LIBRARIES=(0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0)
 
+ENABLED_LIBRARIES[LIBRARY_CPU_FEATURES]=1
```

Comment:
That's true, I considered this, but in the end I decided that such thing should be more explicit. But as you wish…

If we move cpu_features to the end, this will look cleaner, anyways.